environments/magento1/init.env

```diff
@@ -2,8 +2,8 @@
 WARDEN_DB=1
 WARDEN_REDIS=1
 
-MYSQL_DISTRIBUTION=mariadb
-MYSQL_DISTRIBUTION_VERSION=10.3
+DB=mariadb
```

Comment:
@navarr Is this missing the `_DISTRIBUTION` part of the key or is this a weird edge case?

Reply:
Good catch.  Appreciate you reviewing my WIPs.  <3